src/docs/framework-integration/angular.md

```diff
@@ -89,193 +149,292 @@ import { angularOutputTarget } from '@stencil/angular-output-target';
 
 From your Stencil project's directory, run the following command:
 
-```bash
+```bash npm2yarn
 # Link the working directory
 npm link
-# or if using yarn
-yarn link
 ```
 
 From your Angular component library's directory, run the following command:
 
-```bash
+```bash npm2yarn
 # Link the package name
 npm link name-of-your-stencil-package
-# or if using yarn
-yarn link name-of-your-stencil-package
 ```
 
 The name of your Stencil package should match the `name` property from the Stencil component library's `package.json`.
 
-Your component libraries are now linked together. You can make changes in the Stencil component library and run `npm run build` to propagate the changes to the Angular component library.
+Your component libraries are now linked together. You can make changes in the Stencil component library and run `npm run build` to propagate the
+changes to the Angular component library.
 
-> **NOTE**: As an alternative to `npm link` , you can also run `npm install` with a relative path to your Stencil component library. This strategy, however, will modify your `package.json` so it is important to make sure you do not commit those changes.
+:::note
+As an alternative to `npm link` , you can also run `npm install` with a relative path to your Stencil component library. This strategy,
+however, will modify your `package.json` so it is important to make sure you do not commit those changes.
+:::
 
 ## Consumer Usage
 
+### Creating a Consumer Angular App
+
+:::note
+If you already have an Angular app, skip this section.
+:::
+
+From your Angular workspace (`/packages/angular-workspace`), run the following command to generate an Angular application:
+
+```bash
+npx -p @angular/cli ng generate app my-app
+```
+
+### Consuming the Angular Wrapper Components
+
 This section covers how developers consuming your Angular component wrappers will use your package and component wrappers.
 
-If you distributed your components through a primary `NgModule`, developers can simply import that module into their implementation to use your components.
+In order to use the generated component wrappers in the Angular app, you'll first need to build your Angular component library. From the root
+of your Angular workspace (`/packages/angular-workspace`), run the following command:
+
+```bash
+npx -p @angular/cli ng build component-library
+```
+
+Now you can reference your component library as a standard import. If you distributed your components through a primary `NgModule`, you can
+simply import that module into their implementation to use your components.
 
 ```ts
-import { ExampleLibraryModule } from 'your-angular-library-package-name';
+// app.module.ts
+
+import { ComponentLibraryModule } from 'component-library';
 
 @NgModule({
-  imports: [ExampleLibraryModule],
+  imports: [ComponentLibraryModule],
 })
-export class FeatureModule {}
+export class AppModule {}
```

Comment:
When I take this approach, and I put `my-component` in `app.component.html`, it doesn't appear to get registered with the custom elm registry. Does this `app.module.ts` look OK to you?
```
import { NgModule } from '@angular/core';
import { BrowserModule } from '@angular/platform-browser';

import { AppComponent } from './app.component';
import { ComponentLibraryModule } from 'component-library';

@NgModule({
  declarations: [
    AppComponent
  ],
  imports: [
    BrowserModule,
    ComponentLibraryModule
  ],
  providers: [],
  bootstrap: [AppComponent]
})
export class AppModule { }
```

Reply:
@rwaskiewicz It looks like good to me. Did you add the initializer behavior which handles registering the custom elements?: https://github.com/ionic-team/stencil-site/blob/518a8619d865cef0bcd65b27697a78158f05bbb7/src/docs/framework-integration/angular.md?plain=1#L215-L235